react/react-training/src/services/UserService.js

```diff
@@ -1,5 +1,8 @@
 import api from '../config/api';
 
 export default {
-  login: (username, password) => api.post('/Users/login', { username, password })
+  userGet: userId => api.get(`/Users/${userId}`),
```

Comment:
I recommend you to create `loginService` and `userService` to separate responsibilities